src/System.Runtime/tests/System.Runtime.Tests.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="CopyFilesToNonTPADir" BeforeTargets="RunTestsForProject" >
```

Comment:
It will be in the test root directory. What ever you end up doing here we need to be sure it works in our test runs as well and this custom logic will not. cc @MattGal